hangups/ui/__main__.py

```diff
@@ -836,6 +838,40 @@ def main():
         with open(default_config_path, 'a') as cfg:
             cfg.write("")
 
+    # Create empty colors directory if there is none
+    dir_maker(default_colors_path)
+
+    # Read .col files
+    attributes = {'active_tab', 'inactive_tab', 'msg_date', 'msg_sender', 'msg_text', 'status_line', 'tab_background'}
+    for root, dirs, files in os.walk(default_colors_path):
```

Comment:
Seems like a waste to load all the colour schemes when we already know which one the user wants. Can you make it just load the file that's specified?